File: codes/ruby/object_values.rb
require "json"

print("\n", "# JavaScript-like Object.values() in Ruby Hash")

friend = {
    "name" => "Alisa",
    "country" => "Finland",
    "age" => 25
}
print("\n", "friend: ", JSON.pretty_generate(friend, { "indent": " " * 4 }))

print("\n", "friend values: ", JSON.generate(friend.values()).gsub(",", ", "))
# friend values: ["Alisa", "Finland", 25]

print("\n", "friend values: ", JSON.generate(friend.values).gsub(",", ", "))
# friend values: ["Alisa", "Finland", 25]

File: codes/python/object_values.py
import json

print('\n# JavaScript-like Object.values() in Python dictionary')

friend = {
    "name": "Alisa",
    "country": "Finland",
    "age": 25
}
print(f'friend: {json.dumps(friend, indent=4)}')

print(f'friend values: {json.dumps(list(friend.values()))}')
# friend values: ["Alisa", "Finland", 25]
